code/drasil-utils/Utils/Drasil/Concepts.hs

```diff
@@ -348,19 +348,20 @@ a_Gen f t = nounPhrase'' (S "a" +:+ f t) (S "a" +:+ f t) CapFirst CapWords
 -- See 'compoundPhrase' for more on plural behaviour.
 -- /Does not preserve abbreviations/.
 compoundNC :: (NamedIdea a, NamedIdea b) => a -> b -> NamedChunk
-compoundNC t1 t2 = nc
-  (t1 ^. uid ++ t2^.uid) (compoundPhrase (t1 ^. term) (t2 ^. term))
+compoundNC t1 t2 = ncUID
+  ((t1 ^. uid) +++. showUID t2) (compoundPhrase (t1 ^. term) (t2 ^. term))
```

Comment:
Ah, this shows the need for an extra combinator to combine two UIDs. This should then be used here.